src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement/AccessManagementFeatureFlags.cs

```diff
@@ -10,6 +10,21 @@ internal static class AccessManagementFeatureFlags
     /// </summary>
     public const string HostedServicesRegisterSync = $"AccessManagement.HostedServices.RegisterSync";
 
+    /// <summary>
+    /// Specifies if the altinn roles data should streamed from sblbridge service to access management database
+    /// </summary>
+    public const string HostedServicesAllAltinnRoleSync = $"AccessManagement.HostedServices.AllAltinnRoleSync";
```

Comment:
Bør vel høre med Andreas for å få opprettet feature togglene i Azure App config

Reply:
Done